.gitignore

```diff
@@ -134,5 +134,5 @@ plans.lock
 # Edit: include this file now as access to it online requires authorization.
 #scheduler/services/resource/data/telescope_schedules.xlsx
 
-
-
+# Pickled services
+/pickles
```

Comment:
If there are never committed to the repo, at the moment of creating the Docker image the pickles are going to be absent and then in Heroku would basically create everything from scratch, rendering this pointless.

Reply:
Fair enough. I adjusted the file.